test/integration/auth/rbac_test.go

```diff
@@ -574,6 +574,9 @@ func TestRBAC(t *testing.T) {
 			if r.verb == "PATCH" {
 				// For patch operations, use the apply content type
 				req.Header.Add("Content-Type", string(types.ApplyPatchType))
+				q := req.URL.Query()
```

Comment:
Would like to see an integration test that verifies the flag is rejected when not using apply patch and accepted when it is.